l10n_br_base/views/res_partner_view.xml

```diff
@@ -97,7 +101,11 @@
                 </group>
             </xpath>
             <page position="after" name="sales_purchases">
-                <page string="Fiscal" name="fiscal">
+                <page
+                    string="Fiscal"
+                    name="fiscal"
+                    attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                >
```

Comment:
@renatonlima @marcelsavegnago @max3903 @JorgeJuarezCasai

I checked this again an I thing this is OK to hide the fiscal tab IF we set an _inverse method on the country_id so that when set outside from Brazil it sets the fiscal_profile_id to "Não contribuente" (if we don't do that, in Brazil you won't be able to properly set up a foreign customer or supplier).
What do you think?

Reply:
It's not correct hide Fiscal tab in foreign customers because here in brazil if you do a foreign sale you need see fiscal tab. The correct is hide Fiscal tab if company's country is not Brazil.